docs/production-deployment/cloud/api-keys.mdx

```diff
@@ -19,331 +19,362 @@ tags:
   - how-to
 ---
 
-:::tip Support, stability, and dependency info
+:::tip SUPPORT, STABILITY, AND DEPENDENCY INFO
 
-- API Keys used for Cloud Operations Tools (tcld, API, Terraform) are in [Public Preview](/evaluate/development-production-features/release-stages#public-preview)
-- API Keys used for Namespace Authentication are in [Pre Release](/evaluate/development-production-features/release-stages#pre-release)
-  :::
+- API key authentication for Temporal Cloud, `tcld`, SDK-based API calls, and Terraform provider, is currently in [Public Preview](/evaluate/development-production-features/release-stages#public-preview)
+- API key authentication for Namespace authentication is in [Pre-release](/evaluate/development-production-features/release-stages#pre-release)
 
-Temporal Cloud supports secure programmatic access through API Key Authentication.
+:::
 
-## Overview
+Temporal Cloud API key authentication provides secure access to the Temporal Service from your applications.
+API keys are a common industry standard for simple access control.
+They need only one key to authenticate.
+[mTLS certificates](https://docs.temporal.io/references/configuration#tls) offer higher security but they require more complex setup and configuration.
+API keys simplify deployment and management when mTLS is not needed.
 
-Temporal Cloud API Keys provide identity-based authentication where an identity can be either a Temporal User or a [Service Account](/cloud/service-accounts)
-Each key is unique to a single identity and ties directly to their Role-Based Access Control (RBAC) settings, ensuring secure and appropriate access.
+This document introduces Temporal Cloud's API key features:
 
-The process is as follows:
+- [API key overview](#overview)
+- [Global Administrator API key management](#manage-api-keys)
+- [User API key management](#user-api-keys)
+- [Manage API keys for Service Accounts](#serviceaccount-api-keys)
+- [API keys for Namespace authentication](#namespace-authentication)
+- [Use API keys to authenticate](#using-apikeys)
+- [API key best practices](#best-practices)
+- [Troubleshoot your API key use](#troubleshooting)
+- [API keys: Frequently Asked Questions](#faqs)
 
-```text
-API Key (authentication) → Identity (User or Service Account) → RBAC (authorization)
-```
+## API key overview {#overview}
 
-API Keys support 2 main use cases:
+Temporal Cloud API keys offer industry-standard identity-based authentication for Temporal users and [Service Accounts](/cloud/service-accounts).
```

Comment:
This is much better (relative to the above intro) imo -- can we just use this?

Reply:
Why yes. Yes, we can.

Reply:
Deleted the entire intro and replaced with this line.